### DISCUSSION TITLE
What would be needed to make this work with a suspended neovim running in the same shell?

### DISCUSSION THREAD

**Discussion by IndianBoy42** [willothy/flatten.nvim#65] [Ideas]
I used to use CTRL-z a lot, even binding CTRL-z in fish to reopening the suspended neovim, to run some quick commands (files and builds etc). I've sinced changes to using Kitty windows/tabs because of better flatten integration (and other things).

I wonder what would be needed to be able to CTRL-z/:suspend neovim and then be able to open files, pipe commands or do git commits with that background neovim (automatically unsuspending it too) in a as transparent and seamless as possible way

**Comment by willothy** [comment]
I think the best way to do that would probably be rebinding ctrl-z to use a custom function which sets an env var in addition to suspending the process. Then, the same method of checking for host processes can be done as is done with flatten by default. Maybe wrong though, just my first thought.
